bcrhp/src/bcrhp/pages/steps/Step8_SiteClassification.vue

```diff
@@ -253,124 +250,28 @@ onMounted(() => {
+                    @value-updated="updateSelectValue"
+                />
             </div>
             <p class="mb-1">Ownership</p>
-            <div class="card flex flex-wrap gap-4">
-                <div class="flex items-center gap-2">
-                    <RadioButton
-                        v-model="ownership"
-                        inputId="ownership"
-                        value="Not-for-profit"
-                        @change="
-                            valueChanged($event, 'requiredHeritageClassSchema')
-                        "
-                    />
-                    <label for="not_for_profit"> Not-for-profit </label>
-                </div>
-                <div class="flex items-center gap-2">
-                    <RadioButton
-                        v-model="ownership"
-                        inputId="ownership"
-                        value="Private"
-                        @change="
-                            valueChanged($event, 'requiredHeritageClassSchema')
-                        "
-                    />
-                    <label for="private"> Private </label>
-                </div>
-                <div class="flex items-center gap-2">
-                    <RadioButton
-                        v-model="ownership"
-                        inputId="ownership"
-                        value="Public (federal)"
-                        @change="
-                            valueChanged($event, 'requiredHeritageClassSchema')
-                        "
-                    />
-                    <label for="public_federal"> Public (federal) </label>
-                </div>
-                <div class="flex items-center gap-2">
-                    <RadioButton
-                        v-model="ownership"
-                        inputId="ownership"
-                        value="Public (local)"
-                        @change="
-                            valueChanged($event, 'requiredHeritageClassSchema')
-                        "
-                    />
-                    <label for="public_local"> Public (local) </label>
-                </div>
-                <div class="flex items-center gap-2">
-                    <RadioButton
-                        v-model="ownership"
-                        inputId="ownership"
-                        value="Public (provincial)"
-                        @change="
-                            valueChanged($event, 'requiredHeritageClassSchema')
-                        "
-                    />
-                    <label for="public_provincial"> Public (provincial) </label>
-                </div>
+            <div class="flex flex-col">
+                <ConceptRadioButtons
+                    id="ownership"
+                    ref="ownershipField"
+                    v-model="currentHeritageClass.ownership"
+                    graph-slug="heritage_site"
+                    node-alias="heritageClass"
```

Comment:
This should be  ownership not "heritageClass" to match the node value in the graph.